framework/src/controller/controller.js

```diff
@@ -58,29 +49,19 @@ class Controller {
 
 		this.modules = {};
 		this.childrenList = [];
-		this.channel = null; // Channel for controller
 		this.bus = null;
-
-		const storageConfig = config.components.storage;
-		this.storage = createStorageComponent(storageConfig, logger);
-		this.storage.registerEntity('Migration', MigrationEntity);
-		this.storage.registerEntity('NetworkInfo', NetworkInfoEntity);
 	}
 
-	async load(modules, moduleOptions, migrations = {}, networkConfig) {
+	async load(modules, moduleOptions, migrations = {}) {
 		this.logger.info('Loading controller');
 		await this._setupDirectories();
 		await this._validatePidFile();
-		this._initState();
 		await this._setupBus();
-		await this._loadMigrations({ ...migrations, app: controllerMigrations() });
-		await this._initialiseNetwork(networkConfig);
+		await this._loadMigrations({ ...migrations });
 		await this._loadModules(modules, moduleOptions);
 
 		this.logger.debug(this.bus.getEvents(), 'Bus listening to events');
 		this.logger.debug(this.bus.getActions(), 'Bus ready for actions');
-
-		this.channel.publish('app:ready');
```

Comment:
where is this `app:ready` published now?

Reply:
From the application. after loading up the everything including controller.